src/category_theory/isomorphism.lean

```diff
@@ -236,6 +236,7 @@ variables {f g : X ⟶ Y} {h : Y ⟶ Z}
 instance inv_is_iso [is_iso f] : is_iso (inv f) :=
 is_iso.of_iso_inv (as_iso f)
 
+@[priority 900]
```

Comment:
Can you please add a comment explaining why we need this? (1-line summary from the zulip thread?)